addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -23,7 +23,7 @@ var utils = require('web.utils');
 var view_dialogs = require('web.view_dialogs');
 var field_utils = require('web.field_utils');
 var time = require('web.time');
-var ColorpickerDialog = require('web.ColorpickerDialog');
+var ColorpickerDialog = require('web.Colorpicker').ColorpickerDialog;
```

Comment:
For this, I like `const { ColorpickerDialog } = require('web.Colorpicker');`. This avoids importing it under another name making grepping the code less easy.